src/main/resources/org/clulab/asist/grammars/actions.yml

```diff
@@ -173,18 +158,38 @@ rules:
   - name: obstructing
     priority: ${ rulepriority }
     label: Block
+    example: "Some rubble is blocking this door"
     action: removeResearcher
     pattern: |
       trigger = [lemma=/(?i)^(${ block_triggers })/ & tag=/^V/]
-      source: Concept? = >/${agents}/ | <acl [incoming=nsubj]
+      source: Concept? = >/${agents}/
       target: Concept? = >/dobj|acl/
 
+  - name: obstructing2
+    priority: ${ rulepriority }
+    label: Block
+    action: removeResearcher
+    pattern: |
+      trigger = [lemma=/(?i)^(${ block_triggers })/ & tag=/^V/]
+      source: Obstacle? =  <acl
```

Comment:
was the motivation here just to make the above rule easier to read?
Also, did that [incoming=nubj] constraint not need to be there?

Reply:
I had to seperate these into two rules because of a unintended side effect with existential constructions ("There is rubble blocking the door").
Because of the size of the existential extraction, this rule would extract the existential as a source. So I had to seperate these into a generic and a specific rule, to avoid these cases.

The [incoming=nsubj] should not be necessary in the more specific Obstacle context, I believe